spec/features/user_messages_spec.rb

```diff
@@ -0,0 +1,12 @@
+require_relative '../factories/user.rb'
+
+feature 'User post message' do
+
+  scenario 'user can post a message to chitter' do
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)